packages/calling/calling/versionPlugin.js

```diff
@@ -8,9 +8,12 @@ let file;
 
 // eslint-disable-next-line consistent-return
 /**
+ * .
```

Comment:
Would be better to add some descriptions, now that we are public

Reply:
done